### ISSUE TITLE
N N triple bonds in linkers are flagged as undercoordinated

### ISSUE THREAD

**Issue by arosen93** [kjappelbaum/mofchecker#165]
Hey Kevin! I just thought I'd share this one. Seems N-N triple bonds in linkers are flagged as undercoordinated. This might be related to #146. I attached an example for your reference.
[NN_mof.cif.txt](https://github.com/kjappelbaum/mofchecker/files/7140127/NN_mof.cif.txt)

**Comment by kjappelbaum** [comment]
>P.S. I hope you don't mind my out-of-nowhere comments here. I'm not expecting a fix or anything. Just passing on what I find :) 

no, they are really useful :) I introduce so many bugs in my codes that I need many helping hands to find all of them!
